v1/pipeline/steps/cd/k8s-traffic-routing-spec.yaml

```diff
@@ -0,0 +1,31 @@
+title: TrafficRouting
+type: object
+required:
+  - spec
+  - provider
+properties:
+  spec:
+    type: object
+  provider:
+    type: string
+    enum:
+      - SMI
```

Comment:
any other enums are lowercase, any reason why SMI and Istio follow different convention?

Reply:
no particular reason, I just followed tech spec, but you are right I don't see a reason to not keep it as lowercase

Reply:
done